source/containers.tex

```diff
@@ -12958,21 +12963,32 @@
 \rSec2[container.adaptors.general]{In general}
 
 \pnum
-The headers \libheader{queue} and \libheader{stack} define the container adaptors
-\tcode{queue}, \tcode{priority_queue}, and \tcode{stack}.
+The headers \libheader{queue}, \libheader{stack}, and \libheader{flat_map} define the container adaptors
+\tcode{queue}, \tcode{priority_queue}, \tcode{stack}, and \tcode{flat_map}, respectively.
```

Comment:
```suggestion
\tcode{queue}, \tcode{priority_queue}, \tcode{stack}, \tcode{flat_map}, and \tcode{flat_multimap}, respectively.
```

Reply:
Not in the incoming paper. The "respectively" confusion here is pre-existing.